crates/rethnet_eth/src/remote.rs

```diff
@@ -16,7 +16,7 @@ pub mod jsonrpc;
 /// RPC methods
 pub mod methods;
 
-/// helper utilities for use with serde's serialize_with and deserialize_with
+/// helper utilities for use with serde's `serialize_with` and `deserialize_with`
```

Comment:
did clippy catch this or did you? i'd be impressed haha

Reply:
Clippy did  :slight_smile: